apps/src/netsim/NetSimRemoteNodeSelectionPanel.js

```diff
@@ -137,37 +121,59 @@ NetSimRemoteNodeSelectionPanel.prototype.render = function () {
   this.addRouterButton_ = this.getBody().find('#netsim-lobby-add-router');
   this.addRouterButton_.click(this.addRouterCallback_);
 
-  this.connectButton_ = this.getBody().find('#netsim-lobby-connect');
-  this.connectButton_.click(this.connectButtonCallback_);
+  this.getBody().find('.join-button').click(this.onJoinClick_.bind(this));
+  this.getBody().find('.accept-button').click(this.onJoinClick_.bind(this));
```

Comment:
Hm, these callbacks are being passed in on initialization? Do they need to be bound to `this`?